contributors.json

```diff
@@ -6,5 +6,12 @@
             "favourite-music": "https://soundcloud.com/nocopyrightsounds/cartoon-howling-ft-asena-ncs-release",
             "favourite-color": "#fff44f"
         }
+        
+        {
+            "github-username": "ipshitag",
+            "favourite-emoji": "🐧",
+            "favourite-music": "https://soundcloud.com/oof_oofs-oof/lil-bloom-dead",
+            "favourite-color": "#6B6CE6"
+        }
```

Comment:
@ipshitag thanks for the PR 😄

there is a comma `,` missing in this PR. I'll update regarding this in the readme also now.

Reply:
Thankyou @chandan-reddy-k for providing such a beginner friendly project, and being so helpful. I am super duper new but atleast now, I'll forget the ```,```